src/kimp.py

```diff
@@ -1,50 +1,63 @@
 import requests
-import json
+import datetime
 
+#exchange_rate : 6H, upbit_price, binance_price : 1m
 class Kimp:
     upbit_api_url = 'https://api.upbit.com/v1/ticker' #param = markets=
     binance_api_url = 'https://api.binance.com/api/v1/ticker/price' #param = symbol
-    exchange_rate_api_url = ''
-    symbols = ['BTC', 'ETH', 'DOGE']
-    requests_data = requests.get()
+    exchange_rate_api_url = 'https://api.ratesapi.io/api/latest?base=USD&symbols=KRW'
+    requests_data = requests.get(upbit_api_url)
 
-    def error_log(self):
+    def error_log(self, msg):
         pass
 
     def error_check(self):
-        if self.requests_data != 200:
-            print('[ERROR] errorcode : ' + str(self.requests_data.status_code))
+        if self.requests_data.status_code != 200:
+            print(str(datetime.date.ctime) + '[ERROR] errorcode : ' + str(self.requests_data.status_code))
+        
+        return self.requests_data.status_code
     
     def connect_url(self, url = '', params = None):
         if params == None:
             self.requests_data = requests.get(url)
         else:
             self.requests_data = requests.get(url=url, params=params)
         
-        self.error_check()
+        status_code = self.error_check()
+        
+        return status_code
 
-    def get_exchange_rate(self):
-        pass
+    def get_exchange_rate(self): #갱신 주기 6h
+        if (self.connect_url(url=self.exchange_rate_api_url)) == 200:
+            return self.requests_data.json()['rates']['KRW']
+        else:
+            return 0
 
-    def get_binance_price(self, symbol):
+    def get_binance_price(self, symbol='BTC'): #갱신 주기 1m
         symbol = symbol + 'USDT'
         param = {'symbol':symbol}
-        self.connect_url(url=self.binance_api_url, params=param)
-
-        if self.requests_data.status_code == 200:
-            return (int)(self.requests_data.json()['price'])
+        if (self.connect_url(url=self.binance_api_url, params=param)) == 200:
+            return (float)(self.requests_data.json()['price'])
         else:
             return 0
 
-    def get_upbit_price(self, symbol):
+    def get_upbit_price(self, symbol='BTC'): #갱신 주기 1m
         symbol = 'KRW-' + symbol
         param = {'markets':symbol}
-        self.connect_url(url=self.upbit_api_url, params=param)
-
-        if self.requests_data.status_code == 200:
-            return self.requests_data.json()['trade_price']
+        if (self.connect_url(url=self.upbit_api_url, params=param)) == 200:
+            return self.requests_data.json()[0]['trade_price']
         else:
             return 0
 
-    def cal_kimp(self, symbol):
-        pass
\ No newline at end of file
+    def cal_kimp(self, exchange_rate=0, binance_price=0, upbit_price=0): #실행 주기 1m
```

Comment:
```suggestion
    def cal_kimp(self, exchange_rate, binance_price, upbit_price): #실행 주기 1m
```

의미없는 default parameter라고 생각하네요. 0 값은 애초에 들어오면 안되는데 이걸 디폴트 값으로 줄 필요가 있나요?